Sources/SwiftInspectorVisitors/FunctionDeclarationVisitor.swift

```diff
@@ -56,17 +56,21 @@ fileprivate final class FunctionSignatureVisitor: SyntaxVisitor {
       let type = node.type?.typeDescription
     else { return .skipChildren }
 
-    appendArgument(argumentLabelName: argumentLabelName, type: type)
+    let parameterName = node.secondName?.text
+
+    appendArgument(argumentLabelName: argumentLabelName, parameterName: parameterName, type: type)
     return .skipChildren
   }
   override func visit(_ node: ReturnClauseSyntax) -> SyntaxVisitorContinueKind {
     returnType = node.returnType.typeDescription
     return .skipChildren
   }
 
-  fileprivate func appendArgument(argumentLabelName: String, type: TypeDescription) {
+  fileprivate func appendArgument(argumentLabelName: String, parameterName: String?, type: TypeDescription) {
```

Comment:
Technically speaking if a function argument only has a parameter name and no argument label then the single name is the parameter name. The way we've written this function though the single name would be the argument label.

Reply:
Can we rework this function so that the argument label name is optional and the parameter name is required?

Reply:
Oh yeah, I think I got this the other way around. I disagree with making these optional though, that seems incorrect since we'd never really have a nil argument label.

Reply:
I went back and re-read this.

Based on:

> Each function parameter has both an argument label and a parameter name. The argument label is used when calling the function; each argument is written in the function call with its argument label before it. The parameter name is used in the implementation of the function.

I'm again confused. I thought this meant that if we only defined one 👍🏼 
`func foo(bar: String)`

then bar is the argument label _and_ the parameter name. And then 
`func foo(bar bar2: String)`

Reply:
So I don't think 

> Technically speaking if a function argument only has a parameter name and no argument label then the single name is the parameter name

Is correct?

How I understand this is that:

- A _parameter_ has 2 members that _always exist_ an argument label and a name (i.e. the "parameter name")
- When we explicitly define only one "member" in the parameter, then the argument label is equal to the name.

So I think making either of these nil is just incorrect, since they can never technically be nil.

Reply:
Ok, this is what I ended up with https://github.com/fdiaz/SwiftInspector/pull/136/commits/f4c285dd18f85cbd4fd352cb0026d2286ea6e77e